File: src/codes/scattered-concerns/py/uncode.py
import os

class OrderService:
    def create_order(self, order_id: str):
        print(f"[OrderService] 注文作成: {order_id}")  # 直書きログ

        debug_mode = os.environ.get("DEBUG_MODE") == "true"  # 設定の直接参照
        if debug_mode:
            print(f"[DEBUG] 注文詳細: {order_id}")

class UserService:
    def create_user(self, name: str):
        print(f"[UserService] ユーザー作成: {name}")  # 直書きログ

# 実行例
# os.environ["DEBUG_MODE"] = "true"

order_service = OrderService()
order_service.create_order("order-123")

user_service = UserService()
user_service.create_user("hiroshi")

File: src/codes/scattered-concerns/php/uncode.php
<?php

class OrderService {
  public function createOrder(string $orderId): void {
    echo "[OrderService] 注文作成: {$orderId}\n"; // 直書きログ

    $debugMode = getenv("DEBUG_MODE") === "true"; // 設定の直接参照
    if ($debugMode) {
      echo "[DEBUG] 注文詳細: {$orderId}\n";
    }
  }
}

class UserService {
  public function createUser(string $name): void {
    echo "[UserService] ユーザー作成: {$name}\n"; // 直書きログ
  }
}

// 環境変数設定（実行前に CLI などで設定）
// putenv("DEBUG_MODE=true");

$orderService = new OrderService();
$orderService->createOrder("order-123");

$userService = new UserService();
$userService->createUser("hiroshi");
